keras_cv/layers/serializable_sequential.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.api_export import keras_cv_export
+from keras_cv.backend import keras
+from keras_cv.utils.python_utils import classproperty
+
+
+# TODO(tirthasheshpatel): Use `Sequential` model once the bug is resolved.
+# Temporarily substitute the `Sequential` model with this because a
+# bug in Keras/Keras Core prevents the weights of a sequential model to
+# load in TensorFlow if they are saved in JAX/PyTorch and vice versa.
+# This only happens when the `build` method is called in the `__init__`
+# step.
+@keras_cv_export("keras_cv.layers.SerializableSequential")
+class SerializableSequential(keras.layers.Layer):
```

Comment:
Is this still an issue in Keras Core?

Reply:
I ended up removing it since the legacy weights load in all backends in Keras Core and also in TF Keras. I think until some saving issues are addressed, we should just use the legacy weights. Let me know what you both think!